pywbemcli/_cmd_class.py

```diff
@@ -105,14 +114,14 @@ def class_delete(context, classname, **options):
     server. Use this with caution.  It can impact instance providers and
     other components in the server.
 
-    Some server may refuse the operation.
+    Some servers may refuse the operation.
     """
     context.execute_cmd(lambda: cmd_class_delete(context, classname, options))
 
 
 @class_group.command('invokemethod', options_metavar=CMD_OPTS_TXT)
-@click.argument('classname', type=str, metavar='classname', required=True)
-@click.argument('methodname', type=str, metavar='name', required=True)
+@click.argument('CLASSNAME', type=str, metavar='CLASSNAME', required=True,)
```

Comment:
I agree with changing the metavar to upper case, but I would not do that for the actual argument name (first argument). Suggest to change the argument names to lower case again.

Reply:
I thought I did it for all of them.  Will review and make any changes I did not make. I reviewed and all of the changes were made.